src/herbpy/herbrobot.py

```diff
@@ -271,7 +270,72 @@ def SetStiffness(self, stiffness):
         self.head.SetStiffness(stiffness)
         self.left_arm.SetStiffness(stiffness)
         self.right_arm.SetStiffness(stiffness)
-
+        
+    def DetectHuman(self, env, orhuman=2, load_hum=True, 
+                    hrc=False, continuous=True,      
+                    herb_sim=True, enable_legs=True,             
+                    segway_sim=True, action='stamp'):
+        """Use the kinbody detector to detect objects and add
+        them to the environment
+        """
+        import rospy
+        from tf import TransformListener, transformations
+
+        node_name = 'detecthuman'  
+
+        if orhuman==0: 
+            if load_hum==True:
+                #kin 1 - load a skeleton
+                import or_skeletons.load_skeletons as sk
+                humans = []
+                logger.info('Humans_tracking')
+        elif orhuman==1:
+            if load_hum==True:
+                #kin 1- load the human from openrave
+                import humanpy.hum_kin1 as sk
+                humans = []
+                logger.info('Humans_tracking')
+        elif orhuman==2:
+            if load_hum==True:
+                #kin 2 - load the human from openrave
+                import humanpy.hum_kin2 as sk
+                humans = []
+                logger.info('Humans_tracking')
+            if hrc==True:
+                from hrc import assistance                 
+                humans_hrc = []
+                logger.info('HRC')
```

Comment:
What if `orhuman` is not `0`, `1`, or `2`? If I understand this correctly, that is an error and you should throw an exception.